css/fonts.css

```diff
@@ -0,0 +1,40 @@
+
+@font-face {
+ font-family: "Montserrat";
+ src: url("../fonts/MontserratAlternates-Bold.eot"), /* IE9 Compat Modes */
+      url("../fonts/MontserratAlternates-Bold.eot?#iefix") format("embedded-opentype"), /* IE6-IE8 */
+      url("../fonts/MontserratAlternates-Bold.woff") format("woff"), /* Modern Browsers */
+      url("../fonts/MontserratAlternates-Bold.woff2") format("woff2"); /* Modern Browsers */
+ font-weight: bold;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5516260)